src/renderer/containers/FileDownloadDialog.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import { DialogState, State } from '../state';
```

Comment:
Move  imports of external modules to top. Empty line please between imports of external modules and own imports.